ui-ngx/src/app/modules/home/pages/user/user.component.ts

```diff
@@ -56,11 +56,11 @@ export class UserComponent extends EntityComponent<User> {
   }
 
   isUserCredentialsEnabled(): boolean {
-    if (!this.entity || !this.entity.additionalInfo || isUndefined(this.entity.additionalInfo.userCredentialsEnabled)) {
-      return true;
-    } else {
       return this.entity.additionalInfo.userCredentialsEnabled === true;
-    }
+  }
+
+  isUserCredentialPresent(): boolean {
+    return !(!this.entity || !this.entity.additionalInfo || isUndefined(this.entity.additionalInfo.userCredentialsEnabled));
```

Comment:
This is wrong, because difficult to understand. You need to be rewritten using the operator `&&`.

Reply:
fixed